tests/substance_misuse_illsub_seed_coverage.sql

```
@@ -0,0 +1,17 @@
+-- other side of that trade loud: if NHSE adds codes to ILLSUB_COD, it fails and the
+-- seed is re-cut from the refset rather than the default quietly deciding who
+-- counts as a current substance misuser.
+
+SELECT
+    cs.code,
+    cs.code_description
+FROM {{ ref('stg_reference_combined_codesets') }} AS cs
+LEFT JOIN {{ ref('substance_misuse_illsub_status') }} AS s
+    ON cs.code = s.snomed_code
+WHERE UPPER(cs.cluster_id) = 'ILLSUB_COD'
+    AND s.snomed_code IS NULL
```

SQLFluff (4.2.2)

[error] Boolean/null literals must be lower case.

(CP04)